### DISCUSSION TITLE
Vim:E117: Unknow function: floaterm#buffer#getlines

### DISCUSSION THREAD

**Discussion by Kurniawanrzk** [neoclide/coc.nvim#3810] [Q&A] [answered]
### "Vim:E117: Unknow function: floaterm#buffer#getlines"
i get that error, i'm new with nvim so i dont really know how to fix this**

![Screenshot_2022-05-10_23-35-11](https://user-images.githubusercontent.com/92316345/167678969-ce9826c4-6808-4f9e-973c-2575c80cee9e.png)

![Screenshot_2022-05-10_23-35-55](https://user-images.githubusercontent.com/92316345/167678833-0fc5b2b4-6d02-4cee-a711-3c20abf2def7.png)
-----------------------------

**Comment by chemzqm** [comment, marked as answer]
Your coc-floaterm requires other vim plug-in to work please read it’s doc

**Reply by Kurniawanrzk** [reply]
oh silly me, i just relize that, it works now thx man :)
